### DISCUSSION TITLE
having problem on using uc_special_open_if_cf with Driver()

### DISCUSSION THREAD

**Discussion by khuseynov-daymon** [seleniumbase/SeleniumBase#2813] [Q&A] [answered]
Hi, I was using uc_special_open_if_cf without any issues before, 

but recently it has started to give an error like Chrome object does not have it. 

Is there anything changed and we need to use it differently? 

`
from seleniumbase import Driver
driver = Driver(uc=True,
            # headless2=self.headless,  # Headless mode#
            log_cdp=True,  # Log Events
            ad_block_on=True,  # Ad-blocking
            )
driver.uc_special_open_if_cf('http://google.com')
`

Please find attached the error message

![image](https://github.com/seleniumbase/SeleniumBase/assets/108729867/8bd6efa3-6e2d-4ca4-902e-36e7eca57b1c)

**Comment by mdmintz** [comment, marked as answer]
That's an internal-only method. Use one of the actual UC Mode methods for opening a URL:

```python
driver.get(url)  # If UC Mode and site detects bots, then uc_open_with_tab(url)
driver.uc_open(url)  # (Open in same tab with default reconnect_time)
driver.uc_open_with_tab(url)  # (New tab with default reconnect_time)
driver.uc_open_with_reconnect(url, reconnect_time=None)  # (New tab)
```
